v20.1/configure-replication-zones.md

```diff
@@ -24,7 +24,7 @@ For each of the above objects you can control:
 This page explains how replication zones work and how to use the [`CONFIGURE ZONE`](configure-zone.html) statement to manage them.
 
 {{site.data.alerts.callout_info}}
-Currently, only members of the `admin` role can configure replication zones. By default, the `root` user belongs to the `admin` role.
+To configure replication zones, a user must be granted either the [`admin` role](authorization.html#admin-role), [`CREATE` privilege](authorization.html#supported-privileges), or [`ZONECONFIG` privilege](authorization.html#supported-privileges). To configure [`system` objects](#for-system-data), the `admin` role is required.
```

Comment:
I would reword this as `a user must be a member of the admin role`or have been granted ...

Same for the system part, would word it as a user must be a member of the admin role.

A user can be an indirect member of the admin role, ie, a member of a role that is a member of the admin role.

Ditto for the definitions below.

Reply:
TFTR! Done.